sdk/all/src/main/java/io/opentelemetry/sdk/OpenTelemetrySdk.java

```diff
@@ -73,6 +74,11 @@ public Tracer get(String instrumentationName, String instrumentationVersion) {
       return delegate.get(instrumentationName, instrumentationVersion);
     }
 
+    @Override
+    public TracerBuilder tracerBuilder(String instrumentationName) {
+      return delegate.tracerBuilder(instrumentationName);
+    }
```

Comment:
why is this necessary? The default implementation should cover this, precisely, no?

Reply:
`ObfuscatedTracerProvider` derives from the `TracerProvider` interface which has this default impl:

https://github.com/open-telemetry/opentelemetry-java/blob/main/api/all/src/main/java/io/opentelemetry/api/trace/TracerProvider.java#L58-L60

`return DefaultTracerBuilder.getInstance();` is a no-op TracerBuilder.

Reply:
ah. the issue here is in the ObfuscatedTracerProvider, not in this class at all. We should definitely fix the ObfuscatedTracerProvider. This particular class I don't care so much about since it works as its supposed to.

Reply:
This *is* the ObfuscatedTracerProvider which is a static inner class of OpenTelemetrySdk.